Missing Functions on Windows

When I did `go get -u github.com/gotranspile/cxgo/cmd/cxgo` on my windows computer. This is what was happened:
```
go: downloading modernc.org/cc/v3 v3.33.9
# github.com/gotranspile/cxgo/runtime/csys
D:\go\pkg\mod\github.com\gotranspile\cxgo@v0.3.0\runtime\csys\ioctl.go:22:30: not enough arguments in call to syscall.Syscall
D:\go\pkg\mod\github.com\gotranspile\cxgo@v0.3.0\runtime\csys\ioctl.go:22:31: undefined: syscall.SYS_IOCTL
# github.com/gotranspile/cxgo/runtime/cnet
D:\go\pkg\mod\github.com\gotranspile\cxgo@v0.3.0\runtime\cnet\socket.go:52:21: cannot use fd (type int) as type syscall.Handle in argument to syscall.Bind
D:\go\pkg\mod\github.com\gotranspile\cxgo@v0.3.0\runtime\cnet\socket.go:68:2: cannot use fd (type syscall.Handle) as type int in return argument
D:\go\pkg\mod\github.com\gotranspile\cxgo@v0.3.0\runtime\cnet\socket.go:103:30: cannot use fd (type int) as type syscall.Handle in argument to syscall.SetsockoptInt
```
This is because the syscall package is used which is different for each OS.

--- a/runtime/cnet/socket.go
+++ b/runtime/cnet/socket.go
@@ -49,7 +49,7 @@ func Bind(fd int, addr *SockAddr, sz int) int {
 		libc.SetErr(err)
 		return -1
 	}
-	err := syscall.Bind(fd, sa)
+	err := syscall.Bind(Handle(fd), sa)
 	if err != nil {
 		log.Printf("bind(%d, %p, %d): %v", fd, addr, sz, err)
 		libc.SetErr(err)
@@ -65,7 +65,7 @@ func Socket(domain, typ, proto int) int {
 		libc.SetErr(err)
 		return -1
 	}
-	return fd
+	return int(fd)
 }
 
 func Listen(a1, a2 int32) int32 {
@@ -100,7 +100,7 @@ func SetSockOpt(fd int, level int, name int, val *byte, sz int) int {
 			panic(sz)
 		}
 		val := *(*int32)(unsafe.Pointer(val))
-		err = syscall.SetsockoptInt(fd, level, name, int(val))
+		err = syscall.SetsockoptInt(Handle(fd), level, name, int(val))
 	default:
 		log.Printf("setsockopt(%d, %d, %d, %x, %d)", fd, level, name, val, sz)
 		panic("TODO")

--- a/runtime/cnet/types.go
+++ b/runtime/cnet/types.go
@@ -1,0 +1,5 @@
+// +build !windows
+
+package cnet
+
+type Handle = int

--- a/runtime/cnet/types_windows.go
+++ b/runtime/cnet/types_windows.go
@@ -1,0 +1,7 @@
+// +build windows
+
+package cnet
+
+import "syscall"
+
+type Handle = syscall.Handle

--- a/runtime/csys/ioctl.go
+++ b/runtime/csys/ioctl.go
@@ -1,3 +1,5 @@
+// +build !windows
+
 package csys
 
 import (

--- a/runtime/csys/ioctl_windows.go
+++ b/runtime/csys/ioctl_windows.go
@@ -1,0 +1,7 @@
+// +build windows
+
+package csys
+
+func Ioctl(fd uintptr, req uintptr, args ...interface{}) int32 {
+	panic("TODO")
+}

